scripts/main.js

```diff
@@ -0,0 +1,91 @@
+$('#submit').on('click', function () {
+  var authorName = $('#authorName').val().trim();
```

Comment:
Chaining jQuery methods like this works fine, but it makes your code less resilient to DOM changes. It might be good to consider alternative strategies in the future.